R/utils.R

```diff
@@ -143,3 +143,28 @@ summarise_commit <- function(object) {
 
   return(out)
 }
+
+#' @title Load an application and instructions to run shiny.benchmark
+#'
```

Comment:
Can you add more details about what the function will do (create an example application in the provided directory etc.).

Reply:
Thanks for the comment. I will add it on @description